core/services/commonwealth/setup.py

```diff
@@ -0,0 +1,18 @@
+import setuptools
+
+with open("README.md", "r") as readme:
+    long_description = readme.read()
+
+setuptools.setup(
+    name="commonwealth",
+    version="0.1.0",
+    description="Companion library to share common code.",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    license="MIT",
+    packages=setuptools.find_packages(),
+    python_requires=">=3.9",
+    install_requires=[
+        "starlette == 0.13.6",
```

Comment:
missing loguru

Reply:
I had put it there but apparently I removed on the last rebase. Fixing.